.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: woopra-plugin CI
+
+on: 
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
any chance we can use centos?

Reply:
All of the github hosted runners are Ubuntu.  There is a section in the docs about self-hosted runners.  I’ll do a little research on it

Reply:
ok, we can stick to ubuntu for now (probably worth raising this on the internal call)

Reply:
looks like [other repositories](https://github.com/che-incubator/che-dashboard-next/blob/master/.github/workflows/dockerimage.yml) are also using ubuntu, so probably worth talking about